coffee_roaster_market/apps/coffee/tests/test_views.py

```diff
@@ -0,0 +1,252 @@
+import pytest
+import json
+
+from typing import Final, Optional, Iterable
+from django.contrib.auth.models import User
+from rest_framework.test import APIClient
+
+from coffee_roaster_market.apps.coffee.serializers import (
+    SensorialProfilSerializer,
+    GeoLocationSerializer,
+    PlantationSerializer,
+    CoffeeSerializer,
+)
+
+from .factories import (
+    SensorialProfileFactory,
+    GeolocationFactory,
+    PlantationFactory,
+    CoffeeFactory,
+)
+from .assets import parse_json_data
+
+USERNAME: Final[str] = "someuser"
+PASSWORD: Final[str] = "password"
+JSON_FORMAT: Final[str] = "json"
+
+SENSORIAL_PROFILES_URL: Final[str] = "/sensorial_profiles/"
```

Comment:
`from django.urls import reverse` and you can use `reverse(geolocations:list)` afaik.  If some problems occur, probably there is a need to override `get_absolute_url` methods in models. It looks like a additional effort, but it will pay off! 💪

Reply:
Fixed with reverse from `rest_framework.reverse`.